### PR TITLE
VACMS-2388: Remove lightning artifacts from cms.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #2388 
This is the first step - next is to remove modules from composer/ composer.lock.

@swirtSJW - this export munged a bunch of entity_view_display .ymls - I can put them back in, but hoping you could take a look first and see if you have any idea why they are going away.

## Testing done
Spot checks - roles, page layouts on individual nodes, and workflow & Behat

## QA  

- [x] sanned recent log messages for notices and errors related to these changes.  None found.
- [x] Check for loss of roles.  No loss
image
- [x] No loss or change of workflow
image
- [x] Validate lightning modules are disabled (left) 
image

### CODE REVIEW

File: config/sync/user.role.content_admin.yml [hunk 1]
```diff
@@ -1,12 +1,7 @@
 uuid: bfc758cc-8eee-4aed-909c-a5ccbb83f318
 langcode: en
 status: true
-dependencies:
-  module:
-    - lightning_core
-third_party_settings:
-  lightning_core:
-    description: 'For members of the Global UX team, and other content users who have permissions beyond those of Content Publisher. '
```
Comment (swirtSJW) [inline config/sync/user.role.content_admin.yml:9]: Seems odd to be losing these descriptions.  We might need to add them back someplace else in the UI??

File: config/sync/user.role.content_api_consumer.yml [hunk 2]
```diff
@@ -1,12 +1,7 @@
 uuid: b85b2a90-de13-4616-8260-4d90fba7af03
 langcode: en
 status: true
-dependencies:
-  module:
-    - lightning_core
-third_party_settings:
-  lightning_core:
-    description: 'Used by GraphQL to access content via Content API. Should not be given to humans.'
```
Comment (swirtSJW) [inline config/sync/user.role.content_api_consumer.yml:9]: Another description loss to see where it needs to be re-inserted.  .... or maybe that was a lightning thing being able to describe roles??

File: config/sync/user.role.content_creator_benefits_hubs.yml [hunk 3]
```diff
@@ -1,12 +1,7 @@
 uuid: bbd2be72-5d09-4e7d-94c8-ddecbcc1f481
 langcode: en
 status: true
-dependencies:
-  module:
-    - lightning_core
-third_party_settings:
-  lightning_core:
-    description: "Can create benefits hub content (hub landing pages and detail pages).<br/>\r\nUsers with this role should also be editors, reviewers, or publishers."
```
Comment (swirtSJW) [inline config/sync/user.role.content_creator_benefits_hubs.yml:9]: another description loss

File: config/sync/user.role.content_editor.yml [hunk 4]
```diff
@@ -1,12 +1,7 @@
 uuid: dcc05526-22a7-4835-93ec-6d61a476f796
 langcode: en
 status: true
-dependencies:
-  module:
-    - lightning_core
-third_party_settings:
-  lightning_core:
-    description: 'Can create and edit content, but can''t review or publish it. '
```
Comment (swirtSJW) [inline config/sync/user.role.content_editor.yml:9]: another description loss

File: config/sync/user.role.content_publisher.yml [hunk 5]
```diff
@@ -1,12 +1,7 @@
 uuid: 2782bb1f-74f1-4ac9-b2f8-5dd3cc9bfec1
 langcode: en
 status: true
-dependencies:
-  module:
-    - lightning_core
-third_party_settings:
-  lightning_core:
-    description: 'Can publish content.'
```
Comment (swirtSJW) [inline config/sync/user.role.content_publisher.yml:9]: another description loss

File: config/sync/user.role.content_reviewer.yml [hunk 6]
```diff
@@ -1,12 +1,7 @@
 uuid: abeaf14e-b04e-42f4-b009-3f5fa1f03458
 langcode: en
 status: true
-dependencies:
-  module:
-    - lightning_core
-third_party_settings:
-  lightning_core:
-    description: 'Can approve content changes for publishing, but can''t publish it.'
```
Comment (swirtSJW) [inline config/sync/user.role.content_reviewer.yml:9]: another description loss

File: config/sync/user.role.redirect_administrator.yml [hunk 7]
```diff
@@ -1,12 +1,7 @@
 uuid: c342cb58-27f8-4a6a-b698-15b68416f3b7
 langcode: en
 status: true
-dependencies:
-  module:
-    - lightning_core
-third_party_settings:
-  lightning_core:
-    description: 'Can create and edit redirects.'
```
Comment (swirtSJW) [inline config/sync/user.role.redirect_administrator.yml:9]: another description loss

File: config/sync/user.role.vamc_content_creator.yml [hunk 8]
```diff
@@ -1,12 +1,7 @@
 uuid: d61bb4b4-697c-470e-b9ea-47a72730064e
 langcode: en
 status: true
-dependencies:
-  module:
-    - lightning_core
-third_party_settings:
-  lightning_core:
-    description: "Can create content types related to VAMC (except for the parent \"VAMC\" and \"VAMC Operating Status\" content types, which are created by content admins). <br/>\r\nUsers with this role should also be editors, reviewers, or publishers."
```
Comment (swirtSJW) [inline config/sync/user.role.vamc_content_creator.yml:9]: another description loss

File: config/sync/user.role.administrator.yml [hunk 9]
```diff
@@ -1,12 +1,7 @@
 uuid: e917dd41-9d54-42d2-8cdc-7f27d35ccbd6
 langcode: en
 status: true
-dependencies:
-  module:
-    - lightning_core
-third_party_settings:
-  lightning_core:
-    description: 'Has all permissions.'
```
Comment (swirtSJW) [inline config/sync/user.role.administrator.yml:9]: another description loss

File: config/sync/user.role.admnistrator_users.yml [hunk 10]
```diff
@@ -1,12 +1,7 @@
 uuid: f238afd7-c738-4625-a432-2f9eaebf9283
 langcode: en
 status: true
-dependencies:
-  module:
-    - lightning_core
-third_party_settings:
-  lightning_core:
-    description: 'Can create and manage users, roles, and access to sections of content.'
```
Comment (swirtSJW) [inline config/sync/user.role.admnistrator_users.yml:9]: another description loss

File: config/sync/lightning_core.versions.yml [hunk 11]
```diff
@@ -1,245 +0,0 @@
-access_ui: 1.4.0-beta1
```
Comment (swirtSJW) [inline config/sync/lightning_core.versions.yml:1]: I think we need to do a diff between this file and the core.versions yml to see if there was anything defined here that  we are using that is not in the other... which would result in a  module loss.

File: config/sync/core.entity_view_display.paragraph.collapsible_panel.preview.yml [hunk 12]
```diff
@@ -1,29 +0,0 @@
-uuid: 2114a657-f1f4-453b-b0b4-3891e9f2504e
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.paragraph.collapsible_panel.preview.yml:1]: The loss of this concerns me.

File: config/sync/core.entity_view_display.paragraph.link_teaser.user_guides.yml [hunk 13]
```diff
@@ -1,43 +0,0 @@
-uuid: 62d8e14b-5ae6-458d-906e-f9b4b4d4019b
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.paragraph.link_teaser.user_guides.yml:1]: This loss also concerns me

File: config/sync/core.entity_view_display.paragraph.list_of_link_teasers.user_guides.yml [hunk 14]
```diff
@@ -1,40 +0,0 @@
-uuid: c0c4b52a-6f9b-4ffb-9ddd-0145ae25b68f
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.paragraph.list_of_link_teasers.user_guides.yml:1]: Concerned about this loss.

File: config/sync/core.entity_view_display.paragraph.list_of_links.user_guides.yml [hunk 15]
```diff
@@ -1,56 +0,0 @@
-uuid: eedf5602-5332-4aea-9b72-69de665998ce
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.paragraph.list_of_links.user_guides.yml:1]: Concerned about this loss.

File: config/sync/core.entity_view_display.paragraph.lists_of_links.user_guides.yml [hunk 16]
```diff
@@ -1,40 +0,0 @@
-uuid: c1a8a610-826e-43f9-b16c-700b948fa6fc
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.paragraph.lists_of_links.user_guides.yml:1]: Concerned about this loss.

File: config/sync/core.entity_view_display.paragraph.q_a.preview.yml [hunk 17]
```diff
@@ -1,35 +0,0 @@
-uuid: 0910c3fe-57f3-47a5-89f2-7ff36b2a1141
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.paragraph.q_a.preview.yml:1]: Concerned about this loss.

File: config/sync/core.entity_view_display.paragraph.situation_update.preview.yml [hunk 18]
```diff
@@ -1,46 +0,0 @@
-uuid: 7a0ae46e-5067-4d03-a0dc-93b9afd54e33
```
Comment (swirtSJW) [inline config/sync/core.entity_view_display.paragraph.situation_update.preview.yml:1]: Concerned about this loss.

**Comment by ethanteague** [comment]
> There are a lot of losses here that concern me. (unexpected and potentially undesirable.) I think we need to chase some of these down figure out what changes if they go.

Agreed, @swirtSJW - this is gonna be something that we need to carry over into next sprint.

**Comment by swirtSJW** [comment]
@ethanteague  For some reason I can't use the button here to resolve the conflict in the UI.   Can you resolve?

File: docroot/modules/custom/va_gov_backend/va_gov_backend.module [hunk 19]
```diff
@@ -18,6 +17,9 @@ use Drupal\taxonomy\Entity\Term;
 use Symfony\Cmf\Component\Routing\RouteObjectInterface;
 use Symfony\Component\HttpFoundation\RedirectResponse;
 use Drupal\va_gov_build_trigger\Form\PreviewForm;
+use Drupal\Core\Form\FormStateInterface;
```
[the remark this answers is not in the document]
Comment (ethanteague) [reply docroot/modules/custom/va_gov_backend/va_gov_backend.module:20]: Embarrassed to ask this......should I alpha sort based on the class only, or the full use path, @swirtSJW ?

**Comment by ethanteague** [comment]
> @ethanteague For some reason I can't use the button here to resolve the conflict in the UI. Can you resolve?

Done, @swirtSJW :+1:

File: docroot/modules/custom/va_gov_backend/va_gov_backend.module (hunk 19 above)
[the remark this answers is not in the document]
Comment (swirtSJW) [reply docroot/modules/custom/va_gov_backend/va_gov_backend.module:20]: I hilight the whole chunk and ctrl-shift-p in VSCODE and choose "sort lines ascending"

which comes from this extension

Name: Alphabetical Sorter
Id: ue.alphabetical-sorter
Description: Multi line or single line alphabetical sorter.
Version: 2.0.1
Publisher: Ugur Erdal
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ue.alphabetical-sorter

[the remark this answers is not in the document]
Comment (ethanteague) [reply docroot/modules/custom/va_gov_backend/va_gov_backend.module:20]: Awesome. Thanks, @swirtSJW :+1:

**Comment by swirtSJW** [comment]
Hmmm I made this comment a second ago but it disappeared, sorry if it shows twice.

Sorry Ethan It looks like one of the items that was merged yesterday  referenced a lightning config.


image

If you fix it I promise to get this in first thing before anything else.

**Comment by ethanteague** [comment]
> Hmmm I made this comment a second ago but it disappeared, sorry if it shows twice.
> 
> Sorry Ethan It looks like one of the items that was merged yesterday referenced a lightning config.
> 
> image
> 
> If you fix it I promise to get this in first thing before anything else.

No problem a'tall, @swirtSJW - updated :+1:

**Comment by swirtSJW** [comment]
There was no uninstall called in an hook_update_N on this, but in this case it is ok because none of these lightning modules had hook_uninstall()  so they did not show up here
image

**Pull request merged**